app/Http/Controllers/PeacePoemResponses.php

```diff
@@ -18,7 +18,8 @@ public function __invoke()
         $space = $this->spaces->matchingTypeformId(env('PEACEPOEM_TYPEFORM_ID', '0'));
 
         return view('peacePoemResponses', [
-            'responses' => filled($space) ? $this->responses->approvedForSpace($space) : collect(),
+            'responses' => filled($space) ? $this->responses->approvedForSpace($space)->take(100) : collect(),
+            'spaceId' => filled($space) ? $space['id'] : '',
```

Comment:
This could be written [using an optional](https://laravel.com/docs/8.x/helpers#method-optional).

```php
optional($space)->id
```

Which would return `null` if `$space` was null.

FWIW, we usually access eloquent model attributes with object syntax (`$space->id`), not array syntax (`$space['id']`). If you use array syntax, I believe any of your attribute modifier will be ignored.

Reply:
This totally makes sense. Thanks for showing me - I feel like there are so many methods for collections that I still have yet to use.